server/gitrest/src/www.ts

```diff
@@ -61,7 +62,8 @@ winston.configure({
  * Get port from environment and store in Express.
  */
 const port = normalizePort(process.env.PORT || "3000");
-const historian = app.create(provider);
+const externalStorageManager = new ExternalStorageManager(provider.get("externalStorage:endpoint"));
```

Comment:
Is the intent for having this instantiation in `www.ts` to allow for an alternative/custom `ExternalStorageManager` to be passed in? If so, there should probably be an `IExternalStorageManager` to extend (similar to `ISecretManager` in routerlicious' Riddler). Otherwise, it would seem to follow pattern a bit better to move this to `app.ts` next to the instantiation of `RepoManager`.

Reply:
Good idea, I think it makes sense to have an IExternalStorageManager. Thanks for catching